File: Linguagem de Programação I/Extras/Beecrowd_Extra/1047.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int sh = scanner.nextInt();
        int sm = scanner.nextInt();
        int eh = scanner.nextInt();
        int em = scanner.nextInt();
        int st = sh * 60 + sm;
        int et = eh * 60 + em;
        int d = et - st;
        if (d <= 0) {
            d += 24 * 60;
        }
        int h = d / 60;
        int m = d % 60;
        System.out.printf("O JOGO DUROU %d HORA(S) E %d MINUTO(S)\n", h, m);
    }
}

File: Linguagem de Programação I/Extras/Beecrowd_Extra/1047.go
package main

import (
	"fmt"
	"bufio"
	"os"
	"strings"
	"strconv"
)

func main() {
	reader := bufio.NewReader(os.Stdin)
	input, _ := reader.ReadString('\n')
	times := strings.Fields(input)
	sh, _ := strconv.Atoi(times[0])
	sm, _ := strconv.Atoi(times[1])
	eh, _ := strconv.Atoi(times[2])
	em, _ := strconv.Atoi(times[3])

	st := sh * 60 + sm
	et := eh * 60 + em
	d := et - st
	if d <= 0 {
		d += 24 * 60
	}
	h := d / 60
	m := d % 60
	fmt.Printf("O JOGO DUROU %d HORA(S) E %d MINUTO(S)\n", h, m)
}